include/dslash.h

```diff
@@ -86,7 +86,7 @@ namespace quda
 
     bool tuneSharedCarveOut() const override
     {
-      static bool tune_shared = true; // default is to do carve out tuning
+      static bool tune_shared = device::shared_carve_out_supported(); // default is to do carve out tuning
```

Comment:
I'd update the comment to say "the default is to do carve out tuning if the architecture supports it"